src/cloud/musescorecom/musescorecomservice.cpp

```diff
@@ -249,6 +251,63 @@ mu::RetVal<ScoreInfo> MuseScoreComService::downloadScoreInfo(int scoreId)
     return result;
 }
 
+RetVal<ScoresList> MuseScoreComService::downloadScoresList(int pageSize, int pageNumber)
```

Comment:
Can you explain what this method does?
By name, it looks like we get a list of scores, but this method returns a structure with a meta (meta refers to what?) which you are not using

Reply:
The API to retrieve the scores list is a paginated api. So, it will not return all scores at once, but for example the first 20 entries (this is controlled by the pageSize and pageNumber params. 
Because of this, the API also returns some meta info, namely how many pages there are in total etc. (What's returned by the api resembles exactly the ScoresList struct.)
This meta info is used in src/project/view/cloudscoresmodel.cpp, line 138.

Reply:
@cbjeukendrup what do you think about renaming these arguments to use the word "batch" instead of "page" (to avoid possible confusion with UI elements)?

Reply:
Yes, I've renamed the parameters to make them less ambiguous, and also added a comment with extra explanation.